docs/UserGuide.md

```diff
@@ -319,10 +319,10 @@ Format: `oedit INDEX [r/ROOM_NUMBER] [t/ROOM_TYPE] [g/TAG]`
 
 
 Parameters:
-* [INDEX](#index) The index of the room to edit.
-* [ROOM_NUMBER](#room_number) The room number to change the room identified by [INDEX](#index) to.
-* [ROOM_TYPE](#room_type) The room type to change the room identified by [INDEX](#index) to.
-* [TAG](#tag) Optional tags that may be updated for the room identified by [INDEX](#index).
+* [`INDEX`](#index) The index of the room to edit.
+* [`ROOM_NUMBER`](#room_number) The room number to change the room identified by [INDEX](#index) to.
```

Comment:
The `INDEX` at the end of the sentence needs the `` too

Reply:
Done, thanks.